### PR TITLE
Adding controls_ifelse block

### PR DESCRIPTION
**Pull request by AnmAtAnm**
Adding `controls_ifelse`, an if/else block that is loaded from JSON and does not use mutators.

This solves a pending problem on current Android & iOS implementations.
Added this block to the playground simple toolbox and all generators.

Tested across all languages using the playground and [this workspace XML](https://gist.github.com/AnmAtAnm/af905f5838110873e6a607593d9ef85d).

---

This change is [Reviewable](https://reviewable.io/reviews/google/blockly/722)

### CODE REVIEW

**Comment by AnmAtAnm** [comment]
Updated to share code between `controls_if` and `controls_ifelse`. 
Expanded the [set of test blocks I used](https://gist.github.com/AnmAtAnm/af905f5838110873e6a607593d9ef85d) to include more `controls_if` variants, since the implementation is changing.

File: msg/messages.js [hunk 1]
```diff
@@ -239,6 +239,9 @@ Blockly.Msg.CONTROLS_IF_ELSE_TITLE_ELSE = Blockly.Msg.CONTROLS_IF_MSG_ELSE;
 /// tooltip - Describes the 'else' subblock during [https://github.com/google/blockly/wiki/IfElse#block-modification if block modification].
 Blockly.Msg.CONTROLS_IF_ELSE_TOOLTIP = 'Add a final, catch-all condition to the if block.';
 
+/// block text - This is a statement that unary operator that executes the statements in %2 if %1 evaluates to true. Otherwise, the statements in %3 are executed.
```
Comment (NeilFraser) [inline msg/messages.js]: > This is a statement that unary operator that executes...

Huh?  I don't understand this.  Also, non-programer translators will definitely not understand this.  The target audience for this message is something like an English teacher at a school in Hungary.

File: generators/python/logic.js [hunk 2]
```diff
@@ -32,26 +32,49 @@ goog.require('Blockly.Python');
...
+  //   branch = Blockly.Python.statementToCode(block, 'DO' + n) ||
+  //       Blockly.Python.PASS;
+  //   code += 'elif ' + argument + ':\n' + branch;
+  // }
+  // if (block.elseCount_) {
+  //   branch = Blockly.Python.statementToCode(block, 'ELSE') ||
+  //       Blockly.Python.PASS;
+  //   code += 'else:\n' + branch;
+  // }
+  // return code;
```
Comment (NeilFraser) [inline generators/python/logic.js]: Delete this.

File: msg/messages.js (hunk 1 above)
Comment (AnmAtAnm) [inline msg/messages.js]: Sorry. Some copy pasta errors.  Used comments above as basis for rewrite.

File: generators/python/logic.js (hunk 2 above)
Comment (AnmAtAnm) [inline generators/python/logic.js]: Deleted.

**Pull request merged**